metaflow/plugins/kfp/kfp.py

```diff
@@ -645,6 +646,7 @@ def build_kfp_dag(
                     preceding_component_inputs=preceding_component_inputs,
                     preceding_component_outputs=kfp_component.preceding_component_outputs,
                     metaflow_service_url=METADATA_SERVICE_URL,
+                    metaflow_user=METAFLOW_USER,
```

Comment:
What will happen if `METAFLOW_USER==NONE` ? Will it use the default of `metaflow_user: str = "kfp-user",` ?

Reply:
yes